components/icons/icon.js

```diff
@@ -0,0 +1,155 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { loadSvg } from '../../.generated/icons/presetIconLoader.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+export class D2LIcon extends RtlMixin(LitElement) {
```

Comment:
Noticed this while I was putting together my shadyCSS PRs ... looks like `export` can be removed here.